src/components/SidePanel/NavGroup/FilterNavGroup/FilterGroup/index.tsx

```diff
@@ -38,12 +39,25 @@ const FilterGroup: React.FC<FilterGroupProps> = ({
 
   const [newFilterName, setNewFilterName] = useState(name);
   const [isEditing, setIsEditing] = useState(false);
+  const [isValid, setIsValid] = useState(true);
 
   const resetEditState = () => {
     setIsEditing(false);
     setNewFilterName(name);
   };
```

Comment:
oop sorry my bad HAHA I think you also want to reset `isValid` in this 

(problematic scenario: 1. edit filter and type in `*,` 2. press cancel, 3. press edit again and the error message still persists because we forgot to reset `isValid`)

Reply:
Good catch!